apps/src/applab/AppLabCrosshairOverlay.jsx

```diff
@@ -1,5 +1,6 @@
 /** @file App Lab-specific Crosshair Overlay */
 
+import React from 'react';
```

Comment:
I agree with your proposal. Presumably a related step is to tell our linter to stop assuming React exists as a global (such that you need to import React to use it if you dont want linter warnings).